apps/native-component-list/package.json

```diff
@@ -170,7 +170,7 @@
     "expo-modules-tests-core": "^0.7.0",
     "url": "^0.11.0",
     "uuid": "^8.3.0",
-    "victory-native": "^30.4.0"
+    "victory-native": "^36.3.1"
```

Comment:
Is it necessary?

Reply:
Previous version was written using deprecated React lifecycle methods and thus a warning was shown every time I've navigated to the screen consisting of the `victory-native`'s components.